gitter.py

```diff
@@ -190,7 +193,12 @@ def leave(self, reason=None):
 
     @property
     def topic(self):
-        return "TODO"  # TODO
+        json = self._backend.readAPIRequest('rooms', {"q": self.uri}).json()
+        for element in json:
+            if element.uri == self.uri:
```

Comment:
element will always the attribute URI, it is the schema: See Room Schema section https://developer.gitter.im/docs/rooms-resource